http-service-hyper/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+authors = [
+    "Aaron Turon <aturon@mozilla.com>",
```

Comment:
@tirr-c you should add yourself here; if I'm included, then you definitely should be!